exercises/concept/expenses/.docs/instructions.md

```diff
@@ -0,0 +1,115 @@
+# Instructions
+
+Bob is a financial adviser and helps people to manage their expenses. His clients send expenses records and Bob analyses them. Bob has records for the previous periods so that he can see changes in spending.
+
+Bob needs to build a report that contains the top 3 categories of expenses and the total amount of the expenses. Sometimes customers are interested to see expenses in a particular category.
+
+In this exercise, you're going to build a program to help Bob.
+
+All functions receive expenses records and a date period:
+```go
+// Expenses record.
+type Record struct {
+	Date        time.Time
+	Amount      float64
+	Category    string
+}
+
+// Expenses period.
+type DatePeriod struct {
+	From time.Time
+	To   time.Time
+}
+```
+
+The period should include both dates, i.e:
+```go
+p := DarePeriod{From: "2000-10-01", To: "2000-10-31"}
+// p includes "2000-10-01"
+// ...
+// p includes "2000-10-16"
+// ...
+// p includes "2000-10-31"
+//
+// p does not include "2000-09-30"
+// p does not include "2000-11-01"
+```
+
+## 1. Calculate the total amount of expenses in period
+
+Implement the `Total` function to return a sum of expenses in period:
+
+```go
+oct2000 := DatePeriod{From: "2000-10-01", To: "2000-10-31"}
+nov2000 := DatePeriod{From: "2000-11-01", To: "2000-11-30"}
+records := []Record{{Date: "2000-10-11", Amount: 16, Category: "entertainment"}}
+
+Total(records, oct2000)
+// Output: 16
+
+Total(records, nov2000)
+// Output: 0
+```
+
+## 2. Find the top N expenses categories in period
```

Comment:
I think this task should be removed. Adds considerable amount of busy work like sorting, that doesn't necessarily help the student learn more about functions as first-class values.

See the other comments for suggestions on additional tasks. I believe with those additional tasks, this one becomes unnecessary.